include/llvm/IR/IntrinsicsCHERICap.td

```diff
@@ -88,6 +88,9 @@ def int_cheri_cap_type_check :
   GCCBuiltinAlias<"__builtin_mips_cheri_check_type">,
   GCCBuiltin<"__builtin_cheri_type_check">,
   Intrinsic<[], [llvm_fatptr_ty, llvm_fatptr_ty], []>;
+def int_cheri_cap_test_subset:
+  GCCBuiltin<"__builtin_cheri_test_subset">,
```

Comment:
I'm not convinced by the name of this intrinsic or the builtin.  For everything else, we've done noun-verb ordering, so this should be subset_test.